development/app/config/templates/deployment/deployment.yaml

```diff
@@ -62,11 +62,11 @@ spec:
             httpGet:
               path: /healthz
               port: 8080
-            initialDelaySeconds: 3
-            periodSeconds: 2
+            initialDelaySeconds: 1
```

Comment:
Can shorter delay help relieve test failures? Do we see problems due to longer delays?